node/impl/full/eth.go

```diff
@@ -1796,9 +1796,11 @@ func newEthBlockFromFilecoinTipSet(ctx context.Context, ts *types.TipSet, fullTx
 		return ethtypes.EthBlock{}, xerrors.Errorf("failed to compute state: %w", err)
 	}
 
+	skippedTxs := 0
 	for txIdx, msg := range compOutput.Trace {
```

Comment:
```suggestion
	txIdx := 0
	for _, msg := range compOutput.Trace {
```

?

Then just increment the index for each non-skipped message.

Reply:
oh yeah that's better